rts/System/Platform/Watchdog.cpp

```diff
@@ -249,6 +249,7 @@ namespace Watchdog
 		}
 
 		if (0 == --(threadInfo->numreg))
+			// FIXME: undefined behavior, destination object type 'Watchdog::WatchDogThreadInfo' is not TriviallyCopyable [bugprone-undefined-memory-manipulation]
```

Comment:
`*threadInfo = {};`

Reply:
Oh cool, was this an actual UB?

Reply:
memset'ing a std::shared_ptr wasn't kosher, although the control flow prevented any real damage from being done.